src/features/projects/projectsSlice.ts

```diff
@@ -38,9 +38,16 @@ export const getProjectAsync = createAsyncThunk('projects/getProject', async (na
 
 export const updateProjectAsync = createAsyncThunk(
   'projects/updateProject',
-  async ({id, fields}:{id: string, fields: UpdatableProjectFields}): Promise<Project> => {
-    const project = await updateProject(id, fields);
-    return project;
+  async (
+    { id, fields }: { id: string; fields: UpdatableProjectFields },
+    { rejectWithValue }: { rejectWithValue: any },
```

Comment:
I want to avoid to use any, but I cannot handle it.
I referred to
https://redux-toolkit.js.org/api/createAsyncThunk#handling-thunk-errors and
https://redux-toolkit.js.org/usage/usage-with-typescript#createasyncthunk